tests/test_LocalClassifierPerParentNode.py

```diff
@@ -184,3 +186,32 @@ def test_fit_predict():
     lcppn.fit(x, y)
     predictions = lcppn.predict(x)
     assert_array_equal(y, predictions)
+
+
+def test_explainer():
+    rfc = RandomForestClassifier()
+    lcppn = LocalClassifierPerParentNode(
+        local_classifier=rfc,
+    )
+
+    #           a
+    #         /    \
+    #       b       c
+    #      /  \     / \
+    #    d     e   f   g
+
+    X = np.array(
+        [
+            [1, 2, 4],
+            [1, 2, 5],
+            [1, 3, 6],
+            [1, 3, 7],
+        ]
+    )
+    y = np.array([["a", "b", "d"], ["a", "b", "e"], ["a", "c", "f"], ["a", "c", "g"]])
+    X_test = np.array([[1, 2.5, 7]])
+
+    lcppn.fit(X, y)
+    explainer = Explainer(lcppn, data=X, mode="tree")
+    shap_dict = explainer.explain(X_test)
+    assert shap_dict is not None
```

Comment:
I think this is a good test case as it is small  and cool with the ASCII Art!

Nevertheless, what we missed in getting an idea how it works was the `fit` method as currently you do not test this one